src/schema/mutation/addDashboardWithContext.mutation.ts

```diff
@@ -72,26 +88,51 @@ const hasDuplicate = (
   entry: {
     element?: any;
     record?: string | Types.ObjectId;
+    geographic?: string;
   }
 ) => {
   const uniqueEntries = new Set();
-  if (!isNil(get(context, 'resource'))) {
-    for (const item of contentWithContext) {
-      if (get(item, 'record')) {
-        uniqueEntries.add((item as any).record.toString());
+
+  if ('geographic' in entry && 'geography') {
```

Comment:
Hi @RenzoPrats , this is the only thing that I don't get, why set `&& 'geography'` in the condition, that is always true right? Is it necessary?

Besides that looks good to me 👍

Reply:
Hi @unai-reliefapp, indeed it was useless I think I misspelled it, it's already fixed, thank you!